<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #320453, stop:1 #802CBD);

}
QGroupBox#groupBox{
	 border-radius: 80px;
}
  QPushButton#adminButton{
      background-color:red;
    }
QLabel#profileLabel{ 
border-radius: 70px;
 border: 2px solid white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-30</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>34</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ensia library managent system</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Created by : RASLAIN 
 Sofia Charazad</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>411</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#7f00ff;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>34</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ARE YOU............</string>
     </property>
    </widget>
    <widget class="QPushButton" name="studentButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #00C2FF;
    color: white;
    font-weight: bold;
    border-radius: 34px;
    padding: 10px 20px;
}

QPushButton:hover {
    background-color: #2289A9;
}</string>
     </property>
     <property name="text">
      <string>A Student</string>
     </property>
    </widget>
    <widget class="QPushButton" name="adminButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #00C2FF;
    color: white;
    font-weight: bold;
    border-radius: 34px;
    padding: 10px 20px;
}

QPushButton:hover {
    background-color: #2289A9;
}</string>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="profileLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>430</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
